### DISCUSSION TITLE
Remix not support most of the third party libraries

### DISCUSSION THREAD

**Discussion by hencework** [remix-run/remix#8389] [Q&A] [answered]
when i use "react-idle-time" package in my remix project then i got error :
 SyntaxError: Named export 'useIdleTimer' not found. The requested module 'react-idle-timer' is a CommonJS module, which may not support all module.exports as named exports.
[1] CommonJS modules can always be imported via the default export, for example using:
[1] 
[1] import pkg from 'react-idle-timer';
[1] const { useIdleTimer } = pkg;

if import package according to given instruction then code not worked.
i think remix.run not support most of the third party libraries, so i think Next.js is better than remix.run.

**Comment by lili21** [comment, marked as answer]
You can update your remix config to handle this issue.

https://remix.run/docs/en/main/start/v2#servermoduleformat
